AirSkyBoat/AirSkyBoat
# Barge Implementation

**_I affirm:_**

DEBUGGS WILL BE LEFT IN THE CODE UNTIL ITS 100% FINISHED

## What does this pull request do?
Implements the long lost barge.

**What is working?**
- Timekeeper NPCs
- Ticket vendors
- Animations for all the boats (yes it comes and goes)
- Animations for the actual barge

**TODOs:**
- Need to setup routes and cleanup for all the NPCs on the boat (The timekeeper works)
- Time variable for Stubborn Dev (Animation works - yes he jumps!)
- The OOS barge sometimes clips the animation of the 9:20 dock on the southern landing

## Steps to test these changes
!goto carpenters_landing
!pos -300.134 -2.999 505.016  (North Landing)
!pos -148.3136 -2.9521 46.40  (Center Landing)
!pos 210.327 -3.885 -532.511  (South Landing 

Make sure the boat is coming/going at the right time with correct animations.
Make sure vendors work in all circumstances
Make sure timekeeper is working

Closes https://github.com/AirSkyBoat/AirSkyBoat/issues/1102
